data/hai/unfettered jobs.txt

```diff
@@ -0,0 +1,458 @@
+				flee
+	destination "Darkcloak"
+
+
+
+mission "Unfettered Silicon Asteroid Mining"
+	name `Gather silicon from asteroids`
+	description `Mine and then collect 50 tons of silicon from the asteroid field in Wah Yoot and then bring them back to <destination> for <payment>.`
+	job
+	repeat
+	source "Darkcloak"
+	to offer
+		has "First Contact: Unfettered: declined"
+		random < 60
+	on complete
+		payment 250000
+		dialog `The Unfettered unloading the materials you mined seem enthused. "This will allow us to repair our ships!" they tell you as they hand over your payment of <payment> credits.`
+		outfit "Silicon" -50
+
+mission "Unfettered Silver Asteroid Mining"
+	name `Gather silver from asteroids`
+	description `Mine and then collect 15 tons of silver from the asteroid field in Ehma Ti and then bring them back to <destination> for <payment>.`
+	job
+	repeat
+	source "Firelode"
+	destination "Darkcloak"
+	waypoint "Ehma Ti"
+	to offer
+		has "First Contact: Unfettered: declined"
+		random < 60
+	on complete
+		payment 400000
+		dialog `The Unfettered unloading the materials you mined seem satisfied. "This will allow us to repair our ships!" they tell you as they hand over your payment of <payment> credits.`
+		outfit "Silver" -15
+
+
+
+mission "Unfettered Medicals"
+	name `Express medical supplies to <planet>`
+	description `The Unfettered hospitals on Darkcloak are in urgent need of resupply. Collect <tons> of medical supplies from <stopovers>, and bring it to the Unfettered on <destination> by <date> for <payment>.`
+	job
+	repeat
+	deadline 10 1
+	source
+		attributes "unfettered"
+	destination "Darkcloak"
+	to offer
+		has "Unfettered Jump Drive 1: offered"
+		random < 80
+	cargo "medicals" 15 2 .05
+	stopover
+		distance 5 7
+		government "Hai"
+		attributes "spaceport"
+	on visit
+		dialog phrase "generic cargo on visit"
+	on stopover
+		dialog `You are surprised to see the Hai are more than happy to give you the medical supplies needed. "Tell our brothers and sisters to stay strong,"	they ask while loading the supplies on your ship.`
+	on complete
+		payment 10000 4000
+		dialog `The local health specialists seem relieved that you have managed to bring the needed medical supplies back in time.`
+
+
+
+mission "Unfettered Challenge[0]"
+	name `Defeat the young captain`
+	description `A young Unfettered captain has provoked you in a duel. Disable his ship for <payment>.`
+	job
+	repeat
+	source
+		attributes "unfettered"
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		random < 60
+	npc save disable
+		government "Hai (Unfettered Challenge)"
+		personality disables staying heroic target
+		fleet
+			names "hai"
+			variant
+				"Sea Scorpion (Pulse)"
+		conversation
+			action
+				payment 200000
+				"reputation: Hai (Unfettered)" = "reputation: Hai (Unfettered)" + 15
```

Comment:
it is how much you will lose for disabling a crew of 15, but a bit above it to be sure
because if you do smth to a government even if said government doesn't care about it, others will look at it with their values (this gov ignores disabling but UHai gov doesn't)